client/settings/components/shipping-services/group.js

```diff
@@ -56,7 +49,7 @@ const ShippingServiceGroup = ( props ) => {
 	const onChange = ( event ) => updateAll( event, updateValue, services );
 
 	const renderHeader = () => {
-		return <div className="wcc-shipping-services-group-header">
+		return <div className="shipping-services__group-header">
```

Comment:
Do we need `shipping-services__group-header`? - I don't see any CSS rules for it in the code.